### PR TITLE
Task implemented of Namespace methods in wc-admin.php #1734.

### LINKED ISSUE #1734: Namespace methods in wc-admin.php
Background: p90Yrv-Ze-p2

In order to have the plugin published in the .org repo, we need to ensure non-class methods are properly namespaced:

```
## Generic function (and/or define) names 

All plugins must have unique function names, namespaces, defines, and classnames. This prevents your plugin from conflicting with other plugins or themes. We need you to update your plugin to use more unique and distinct names.
```

I believe most of the offending function names exist in `wc-admin.php`, so thinking we should standardize on the `wc_admin_` prefix:

## wc-admin.php

Existing Method | New Name
-----------------|------------
`dependencies_satisfied` | `wc_admin_dependencies_satisfied`
`do_wc_admin_daily` | `wc_admin_do_daily_notes`
`activate_wc_admin_plugin` | `wc_admin_activate_plugin`
`possibly_deactivate_wc_admin_plugin` | `wc_admin_possibly_deactivate_plugin`
`deactivate_wc_admin_plugin` | `wc_admin_deactivate_plugin`

## lib/client-assets.php

Existing Method | New Name
-----------------|------------
`format_order_statuses` | `wc_admin_format_order_statuses`

## lib/admin.php

Existing Method | New Name
-----------------|------------
`woocommerce_embed_page_header` | `wc_admin_embed_page_header`

### PR DESCRIPTION
**Pull request by ronakganatra9**
Fixes #1734 

As per mentioned in the details I have changed the namespace in wc-admin.php and on /lib/client-assets.php and lib/admin.php

### CODE REVIEW

**Comment by timmyc** [comment]
@ronakganatra9 thanks very much for the PR 🙇

File: lib/admin.php [hunk 1]
```diff
@@ -381,7 +381,7 @@ function wc_admin_output_breadcrumb( $section ) {
  *
  * @todo Icon Placeholders for the ActivityPanel, when we implement the new designs.
  */
-function woocommerce_embed_page_header() {
+function wc_admin_woocommerce_embed_page_header() {
```
Comment (psealock) [inline lib/admin.php]: We can remove the `woocommerce` portion of the prefix, so the name will be `wc_admin_embed_page_header`.

File: lib/client-assets.php [hunk 2]
```diff
@@ -133,7 +133,7 @@ function wc_admin_register_script() {
  * @param array $statuses Order statuses.
  * @return array formatted statuses.
  */
-function format_order_statuses( $statuses ) {
```
Comment (timmyc) [inline lib/client-assets.php]: Looks like there are a few more spots where `format_order_statuses()` is being used:

```
lib/admin.php
523:		'options'     => format_order_statuses( wc_get_order_statuses() ),
532:		'options'     => format_order_statuses( wc_get_order_statuses() ),
```

**Review by timmyc — COMMENTED** [review]
Thanks so much for the PR, looks like just a few more spots to update on the usage of `format_order_statuses`

**Review by psealock — COMMENTED** [review]
Thanks for the pull request @ronakganatra9 !

I made a comment on one function name that isn't quite right, and @timmyc's comment too but otherwise its looking good.

File: lib/admin.php (hunk 1 above)
Comment (ronakganatra9) [reply lib/admin.php]: Hi Paul,

Thank you for your thoughts, I have updated my PR with the changes. Please review it and let me know your feedback on the same.

Thanks,
Ronak

**Comment by ronakganatra9** [comment]
> Thanks for the pull request @ronakganatra9 !
> 
> I made a comment on one function name that isn't quite right, and @timmyc's comment too but otherwise its looking good.

Hi Paul,

Thank you for your thoughts, I have updated my PR with the changes. Please review it and let me know your feedback on the same.

Thanks,
Ronak

File: lib/client-assets.php (hunk 2 above)
Comment (ronakganatra9) [reply lib/client-assets.php]: Hi Timmy,

Thank you for reviewing my PR and sharing your feedback. I have implemented the changes and updated my PR. Please review it and share your feedback on it.

Thanks & Regards,
Ronak

**Comment by psealock** [comment]
Changes are looking great @ronakganatra9 ! 

It looks like an extra file got checked in (DEADJOE) however. If you can remove that, this one looks good to be merged. If you could [rebase](https://www.atlassian.com/git/tutorials/rewriting-history/git-rebase) as well to keep the commit history clean, that would be great too. Thanks.

**Comment by ronakganatra9** [comment]
> Changes are looking great @ronakganatra9 !
> 
> It looks like an extra file got checked in (DEADJOE) however. If you can remove that, this one looks good to be merged. If you could [rebase](https://www.atlassian.com/git/tutorials/rewriting-history/git-rebase) as well to keep the commit history clean, that would be great too. Thanks.

Hi @psealock , Thanks for reviewing the changes, I have removed the extra field and applied git rebase also. Let me know if anything else.

**Review by timmyc — APPROVED** [review]
Looking good and testing well, thanks again @ronakganatra9

**Comment by timmyc** [comment]
@ronakganatra9 looks like one more rebase and this will be good to go

**Comment by psealock** [comment]
@ronakganatra9 you'll need to rebase from master to combine all your commits into one and resolve the merge conflict.

```
git rebase -i master
```
If you run into problems, I'm happy to help out.

**Comment by ronakganatra9** [comment]
Hello @psealock ,

Thanks for suggesting me the command and helping me with this. I have applied the command which you suggested it and I got below message.

No modified files, so no updates needed.
Successfully rebased and updated detached HEAD.

See the screenshots and let me know if I am doing anything wrong.

Sorry for taking long for this.

git-rebase
Screenshot from 2019-03-14 10-39-33

**Comment by ronakganatra9** [comment]
Thank you @timmyc  for reviewing the PR.

**Review by joshuatf — COMMENTED** [review]
@ronakganatra9 

>No modified files, so no updates needed.
Successfully rebased and updated detached HEAD.

I think things are looking okay now and I'm not seeing any conflicts, but a quick command if you need to pull updates on a fork and rebase in the future - `git fetch upstream && git rebase upstream/master`

We may also want to drop `package-lock.json` changes from this PR if you could do that.

>Sorry for taking long for this

No problem 😄 - Thanks for taking the time to open up this PR!

**Comment by ronakganatra9** [comment]
Hi @joshuatf ,

Thank you for the command. I have dropped the package-lock.json file.

Thanks,
Ronak

**Comment by jeffstieler** [comment]
Hi @ronakganatra9 - thanks again for taking the time to contribute.

I've cherry picked some commits from this PR and created a new one to keep the git history clean: https://github.com/woocommerce/woocommerce-admin/pull/1804

Please check that one out to make sure it's faithful to the changes you made.

---

Here's some feedback on the last changes made:

It seems the `DEADJOE` file has returned. It can help to explicitly stage specific files to each commit, that way you know what's getting added. Are you perhaps using a command like `git add -A` or similar?

Also, @joshuatf was requesting that you remove the _changes_ you made to the `package-lock.json` file, not remove the file entirely. Sorry for the confusion.

**Comment by timmyc** [comment]
Closing this out since the changes got cherry-picked and merged in #1804

**Pull request closed without merging**